tests/integration/ini/test_transaction_tracer_max_segments_no_cap.php

```diff
@@ -82,7 +82,7 @@
 
 
 function my_function(){
-    printf('');
+    time_nanosleep(0, 50000); // force non-zero duration for the segment not to be dropped, 50usec should be enough
```

Comment:
Was this function being dropped even though we were custom tracing it?

Reply:
Yes, it can get (and sometimes was getting) dropped because of this condition: https://github.com/newrelic/newrelic-php-agent/blob/1563e8045968fe0c6ee228c566ca080afceba6c1/axiom/nr_segment_traces.c#L371-L378. Agent's time resolution is microseconds.